parsec/api/data/certif.py

```diff
@@ -0,0 +1,72 @@
+# Parsec Cloud (https://parsec.cloud) Copyright (c) AGPLv3 2019 Scille SAS
+
+from typing import Optional
+from uuid import UUID
+
+from parsec.types import DeviceID, UserID
+from parsec.crypto_types import VerifyKey, PublicKey
+from parsec.serde import fields, post_load
+from parsec.api.protocol.realm import RealmRole, RealmRoleField
+from parsec.api.data.base import BaseSignedData, BaseSignedDataSchema
+
+
+class UserCertificateContent(BaseSignedData):
+    class SCHEMA_CLS(BaseSignedDataSchema):
```

Comment:
Are we sure about the `SCHEMA_CLS` naming?

Reply:
the idea was to have
```python
class UserCertificateContentSchema(BaseSignedDataSchema):
    [...]

class UserCertificateContent(BaseSignedData):
    SCHEMA_CLS = UserCertificateContentSchema
    [...]
```
but it quickly end up mixed together since schema alone is not useful

I can replace `SCHEMA _CLS` by `Schema` (or `SchemaCls`), any preference ?

Reply:
`Schema` sounds good!